tests/js/client/load-balancing/load-balancing-pregel-auth-cluster.js

```diff
@@ -138,39 +137,50 @@ function PregelAuthSuite () {
           resultField: "result"
         }
       };
-      let result = sendRequest(users[0], 'POST', url, task, true);
-
-      assertFalse(result === undefined || result === {});
-      assertEqual(result.status, 200);
-      assertTrue(result.body > 1);
-
-      const taskId = result.body;
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[0], 'GET', url, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertEqual(result.status, 200);
-      assertTrue(result.body.state === "loading" || result.body.state === "running");
-      // check if we've updated the persisted document state properly
-      let persistedState = db[pregelSystemCollection].document(taskId);
-      assertTrue(persistedState.data.state === "loading" || persistedState.data.state === "running");
-
-      require('internal').wait(5.0, false);
-
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[0], 'DELETE', url, {}, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertFalse(result.body.error);
-      assertEqual(result.status, 200);
-
-      // check if we've updated the persisted document state properly after deletion
-      persistedState = db[pregelSystemCollection].document(taskId);
-      assertTrue(persistedState.data.state === "canceled");
+      const postResponse = sendRequest(users[0], 'POST', baseUrl, task, true);
+      assertFalse(postResponse === undefined || postResponse === {});
+      assertEqual(postResponse.status, 200);
+      const pid = postResponse.body;
+      assertTrue(pid > 1);
+
+      // check that pregel status of run can be retrieved
+      // need to wait for short time until pregel run was created
+      const sleepIntervalSeconds = 0.2;
+      const maxWaitSeconds = 1;
```

Comment:
As this is a maximum for waiting do we want to be more generous here?
It is supposed to detect dead-locks/logic errors and will fail on very high timeouts.
Now we could get false negatives on slow machines.
Also on typical runs we should get nowhere near the timeout.

Therefore I would suggest a 10s max wait.
WDYT?